rpm/Makefile

```diff
@@ -40,20 +40,24 @@ fedora: fedora-28 fedora-27 fedora-26 ## build all fedora rpm packages
 .PHONY: centos
 centos: centos-7 ## build all centos rpm packages
 
+engine-$(ARCH).tar:
+	$(MAKE) -C ../image image-linux
+	docker save $(shell cat ../image/image-linux) > $@
```

Comment:
Should be changed to `docker save $$(cat ../image/image-linux)`

Reply:
Did the same in deb/Makefile